Rubberduck.Parsing/VBA/AttributesUpdater.cs

```diff
@@ -0,0 +1,176 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using Antlr4.Runtime;
+using Antlr4.Runtime.Misc;
+using Rubberduck.Parsing.Grammar;
+using Rubberduck.Parsing.Rewriter;
+using Rubberduck.Parsing.Symbols;
+using Rubberduck.Parsing.VBA.Parsing;
+
+namespace Rubberduck.Parsing.VBA
+{
+    public class AttributesUpdater : IAttributesUpdater
+    {
+        private readonly IParseTreeProvider _parseTreeProvider;
+
+        public AttributesUpdater(IParseTreeProvider parseTreeProvider)
+        {
+            _parseTreeProvider = parseTreeProvider;
+        }
+
+
+        public void AddAttribute(IRewriteSession rewriteSession, Declaration declaration, string attribute, IReadOnlyList<string> values)
+        {
+            Debug.Assert(rewriteSession.TargetCodeKind == CodeKind.AttributesCode);
```

Comment:
Logging this might be better than the assert.

Reply:
I used an assert because it should definitely be caught on debug builds.

I will replace it with logging and a bailout.